### ISSUE TITLE
Accordion component doesnot allow to override the focus outline color

### ISSUE THREAD

**Issue by MainakHCL** [HCL-TECH-SOFTWARE/enchanted-react-components#282]
### Latest version

- [x] I have tested the latest version

### Steps to reproduce

Steps:
1. import <Accordion /> from anchanted.
2. Try to override the colors of outline when Accordion is focused via keyboard.
3. Accordion outline colors are not getting overridden due to javascript way of executing and adding outline colors.

### Current behavior

Accordion outline colors cannot be overridden when focused via keyboard.

### Expected behavior

Accordion outline colors should be allowed to override when focused via keyboard.

### Context

We need to change the outline color of accordion to support dark mode. The primary.main color is not accessible in dark mode and fails color contrast ratio.
